include/functions.h

```diff
@@ -1382,18 +1321,20 @@ s16 func_800DFCDC(Camera* camera);
 // UNK_TYPE4 func_800DFF44(void);
 s16 Camera_SetFlags(Camera* iParm1, s16 flags);
 s16 Camera_ClearFlags(Camera* camera, s16 flags);
-// void func_800DFFAC(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE2 param_5,
-// UNK_TYPE2 param_6, UNK_TYPE2 param_7); UNK_TYPE4 func_800E007C(Camera* param_1, Camera* param_2); UNK_TYPE4
-// func_800E01AC(void); void func_800E01B8(void); void func_800E01DC(s32 param_1, u32 param_2, UNK_TYPE4 param_3,
-// UNK_TYPE4 param_4, UNK_TYPE2 param_5, UNK_TYPE2 param_6); UNK_TYPE4 func_800E0228(void); void func_800E0238(void);
+// void func_800DFFAC(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE2 param_5, UNK_TYPE2 param_6, UNK_TYPE2 param_7);
+// UNK_TYPE4 func_800E007C(Camera* param_1, Camera* param_2);
+// UNK_TYPE4 func_800E01AC(void);
+// void func_800E01B8(void);
+// void func_800E01DC(s32 param_1, u32 param_2, UNK_TYPE4 param_3, UNK_TYPE4 param_4, UNK_TYPE2 param_5, UNK_TYPE2 param_6);
+// UNK_TYPE4 func_800E0228(void);
+// void func_800E0238(void);
 void func_800E02AC(Camera* camera, Actor* actor);
 // void func_800E0308(void);
 // void func_800E031C(void);
 void func_800E0348(Camera* camera);
-DamageTable* DamageTable_Get(s32 index);          // func_800E03A0
+DamageTable* DamageTable_Get(s32 index); // func_800E03A0
 void DamageTable_Clear(DamageTable* damageTable); // func_800E03CC
```

Comment:
```suggestion
DamageTable* DamageTable_Get(s32 index);
void DamageTable_Clear(DamageTable* damageTable);
```
I don't think these comments are necessary anymore, and since your already adjusting the file.